apl/tabular-operators/join-operator.mdx

```diff
@@ -0,0 +1,175 @@
+| `rightouter`       | All rows from the right, with matching rows from the left or `null`  | Right-dominant                 |
+| `fullouter`        | All rows from both tables, with unmatched rows padded with `null`    | Complete join                  |
+| `leftanti`         | Rows in the left table with no matches in the right table            | No matches                     |
+| `rightanti`        | Rows in the right table with no matches in the left table            | No matches                     |
+| `leftsemi`         | Rows in the left table with at least one match in the right table    | Matching rows (left table only)|
+| `rightsemi`        | Rows in the right table with at least one match in the left table    | Matching rows (right table only)|
+
+### Choose the right kind of join
+
+- Use `inner` for standard joins where you need all matches.
+- Use `leftouter` or `rightouter` when you need to retain all rows from one table.
+- Use `leftanti` or `rightanti` to find rows that do not match.
+- Use `fullouter` for complete combinations of both tables.
+- Use `leftsemi` or `rightsemi` to filter rows based on existence in another table.
+
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+The `join` operator in APL works similarly to the `join` command in Splunk SPL. However, APL provides additional flexibility by supporting various join types (e.g., `inner`, `outer`, `leftouter`). Splunk uses a single default join type.
+
+<CodeGroup>
+```sql Splunk example
+index=logs | join type=inner [search index=traces]
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| join kind=inner ['otel-demo-traces'] on $left.id == $right.trace_id
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+The `join` operator in APL resembles SQL joins but uses distinct syntax. SQL uses `FROM` and `ON` clauses, whereas APL uses the `join` operator with explicit `kind` and `on` clauses.
+
+<CodeGroup>
+```sql SQL example
+SELECT *
+FROM logs
+JOIN traces
+ON logs.id = traces.trace_id
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| join kind=inner ['otel-demo-traces'] on $left.id == $right.trace_id
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
+
+```kusto
+LeftDataset
+| join kind=KindOfJoin RightDataset on Conditions
+```
+
+### Parameters
+
+- `LeftDataset`: The first dataset, also known as the outer table or the left side of the join. If you expect one of the datasets to contain consistently less data than the other, specify the smaller dataset as the left side of the join.
+- `RightDataset`: The second dataset, also known as the inner table or the right side of the join.
+- `KindOfJoin`: Optionally, the [kind of join](#kinds-of-join) to perform. The only supported value and the default is `inner`.
```

Comment:
Probably can leave off supported here, worried we'll forget to update it :-) Let them click the link to learn more there.